### ISSUE TITLE
CAM bug on raspian armhf

### ISSUE THREAD

**Issue by chessplayer60** [mpmc/tvheadend#89]
Hi Mark,

first of all, thanks for providing the unofficial [Bintray repo](https://dl.bintray.com/mpmc/deb) for **armhf**. Since they say to contact you in case of problems in the [TVHeadend Wiki](https://tvheadend.org/projects/tvheadend/wiki/AptRepository), this is what I am doing. I would like to make you aware of the following [Bug Repport](https://tvheadend.org/issues/4888) (see especially notes [15](https://tvheadend.org/issues/4888#note-15) and my own [21](https://tvheadend.org/issues/4888#note-21)). Unfortunately, the problem does not seem to be addressed.

What the notes mentioned above show is, that somewhere between versions 4.3-734 and 4.3-1006 CAM support got broken for armhf and this is stll the case with your latest 4.3-1288 from the unstable raspbianstretch repo. Myself, I have a working system with 4.3-632, which I got from rasbianstretch a while back, where now there isn't anything to be found anymore, unfortunately. Also, the stable 4.2.6 version from your stable repo works (but does not have the cute icons showing the content type ...).

Would be great if you had a way to do something about this or at least make someone who can aware of the issue.

Cheers,

chessplayer

**Comment by mpmc** [comment]
Sorry to hear you're having issues. I suspect this isn't an issue for just armhf but for the rest too. Unfortunately I'm just the guy who provides the builds for armhf. I'll do my best to help however. 

Out of interest have you tried a clean config? Does it still crash at all?

**Comment by chessplayer60** [comment]
Hi, not sure what you mean by clean config. The Armbian was brand new, but the TVH config was migrated from a 4.2 version on an RPi with LibreELEC. But that was what I did for the cubietruck running the 4.3-632 as well and it was not an issue. Also, until I activated the CAM, TVH also worked.

I can think of one way you might be able to help. I have no idea how much effort that would be, so I am just putting it out there. Maybe you could provide older 4.3 builds (between 4.3-734 and 4.3-1006) in some kind of testing-repo on bintray. I would be willing to try those out on the cubietruck I am running the working version on (with the new Armbian, of course) and maybe then we could pinpoint the first version where things got broken.

**Comment by mpmc** [comment]
by clean config, I mean not from a backup, from a "new" install (delete /home/hts/.hts/tvheadend).

You can find raspbianstretch debs here: https://doozer.io/tvheadend/tvheadend/builds

**Comment by chessplayer60** [comment]
Hi, yes, that is what I thought. So, I did this using your bintray repo for raspbianstretch, which has version 4.3-1288 as the primary candidate, which was installed.
So, here is what happened: without using the CAM, everything was fine. Then I added the CAM (newcamd) and was at least able to save that in the webif, where the green arrow was shown for a little while. Then, TVH crashed. After restarting, it took a few seconds before TVH crashed again. 
So, I removed the file from the caclient-subfolder which specified the newcamd and everything worked again (webif + streaming to vlc vis playlist).

I will now try downloading the debs from doozer to find the latest working version ...
